SEED-platform/seed
# Fix first_five_rows method to be more flexible

#### Any background context you want to provide?
If an excel file has carriage returns in a cell, and then a blank line, then the first five rows would not parse correctly. The mapping screen would show more than 5 columns and the data alignment was incorrect.

#### What's this PR do?
The new first_five_rows code splits the first_five_rows string by crlf, then appends each line until the total number of columns matches up.

#### How should this be manually tested?

* Import the new test file, seed/tests/data/portfolio-manager-sample-with-crlf.xlsx
* Make sure that the first_five_rows api request returns only 5 rows
* Verify that the mapping page looks correct

#### What are the relevant tickets?
#1689 

#### Screenshots (if appropriate)

#### Definition of Done:
